### DISCUSSION TITLE
Help Request - Brave Browser

### DISCUSSION THREAD

**Discussion by BraneStawm23** [DNSCrypt/dnscrypt-proxy#1756] [Q&A]
Hello

I have been using dnscrypt-proxy for well over a year with absolutely no issues and am very pleased with it. Unfortunately I had to install Braver Browser recently (firefox is my default) and it completely bypasses my block list. Brave will not accept 'https://127.0.0.1:3000/dns-query' as my dns server. 

I have scanned to interwebs and cannot find any answers. Could someone please advise as to what am I doing wrong

Many thanks

**Comment by lifenjoiner** [comment]
I don't use Brave browser. I think you should dig out whether Brave has this feature first ... and then, if it support DoH, what's the requirement ... You'd better see what's Brave's document saying ...
